ci/scripts/driver.sh

```diff
@@ -86,11 +88,19 @@ for pr in ${pr_list}; do
     done
     rm -Rf "${pr_dir}"
   fi
+  # Check to see if this PR is Labeled CI-PR-Cases to get cases from itself
```

Comment:
Is there any reason to not just always pull cases from the PR?

Reply:
Not sure I understand the question, the PR is pulled when it is built.

Reply:
Right, but you are relying on whether there is a label set to use the cases from the PR rather than what you already have. Why not always use the ones in the PR? That eliminates the need to set yet another label, and also avoids problems caused by tests being added or changed and the label not being set.

Reply:
Oh, I understand the question now, yes I guess we could just always get cases from the PR, I'm going to refer this question to Raul.

Reply:
The problem I was told is that users can then override the intent of the tests by leaving them out, but like you said, that would be obvious to you.